test/ArrayFieldTemplate_test.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+
+import {expect} from "chai";
+import {createFormComponent, createSandbox} from "./test_utils";
+
+describe("ArrayFieldTemplate", () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("Custom ArrayFieldTemplate of string array", () => {
+
+    function ArrayFieldTemplate(props) {
+      return (
+        <div className="custom-array">
+          {props.canAdd && <button className="custom-array-add"></button>}
+          {props.items.map(element => {
+            return (
+              <div className="custom-array-item" key={element.index}>
+                {element.hasMoveUp &&
+                  <button className="custom-array-item-move-up"></button>}
+                {element.hasMoveDown &&
+                  <button className="custom-array-item-move-down"></button>}
+
+                {element.children}
+              </div>
+            );
+          })}
+        </div>
+      );
+    }
+
+    const formData = ["one", "two", "three"];
+
+    describe("not fixed items", () => {
+      const schema = {
+        type: "array",
+        title: "my list",
+        description: "my description",
+        items: {type: "string"}
+      };
+
+      const {node} = createFormComponent({
```

Comment:
Please move this to a `beforeEach` statement, so the element is not created as soon as the `describe` call is executed. Created troubles in the past. Please refer to existing other tests to see how it's done.